includes/migration/wc/class-fs-wc-migration-endpoint.php

```diff
@@ -22,6 +23,27 @@ class FS_WC_Migration_Endpoint extends FS_Migration_Endpoint_Abstract {
 		 */
 		private static $_instance;
 
+		/**
+		 * @var stdClass Order data Example in /tests/order.txt
+		 */
+		private $_order;
+
+		/**
+		 * @param string $name Property to return
+		 * @return mixed|null Value if property exists else null
+		 */
+		public function __get( $name ) {
```

Comment:
In general, if you want to expose specific params, the best practice is adding the relevant getters. In the current context, if you read my comment about the dependency of `FS_WC_Migration` and `FS_WC_Migration_Endpoint` (that should be removed), I think you can get rid of this code at all.